src/Resources/MSGraph.Autorest/custom/Get-AzADApplicationFederatedIdentityCredential.ps1

```diff
@@ -0,0 +1,163 @@
+
+# ----------------------------------------------------------------------------------
```

Comment:
If Application is considered as parent object of FederatedIdentity, autorest doesn't support pipeline parameter object. We don't need this one for the time being. What's your thought?

Reply:
I'll say it's going to be very common that customer might want to pipe an application into federated identity cmdlets, we can remove these when autorest have them supported but I suggest keep them for now

Reply:
OK